DyamicLinking.md

```diff
@@ -0,0 +1,34 @@
+# Dynamic linking
+
+Dynamic loading of code is in [the MVP](MVP.md) in the form of
+[modules](Modules.md), but all loaded modules have their own separate
+[linear memory](AstSemantics.md#linear-memory) and cannot share
+[function pointers](ASTSemantics.md#calls). Dynamic linking will allow
+developers to share memory and function pointers between WebAssembly modules.
+
+WebAssembly will support both load-time and run-time (`dlopen`) dynamic linking
+of both WebAssembly modules and non-WebAssembly modules (e.g., on the Web, ES6
+ones containing JavaScript).
```

Comment:
Wow, I see this is pre-existing, but I think this para is totally wrong: we've only ever discussed dynamically linking wasm modules; I'm not even sure what dynamically linking an ES6 module would mean.